crates/bevy_render/src/render_graph/nodes/pass_node.rs

```diff
@@ -37,7 +37,7 @@ pub struct PassNode<Q: WorldQuery> {
 
 impl<Q: WorldQuery> fmt::Debug for PassNode<Q> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("PassNose")
+        f.debug_struct("PassNode")
```

Comment:
PassNose :rofl:

Reply:
I know! I had a good chuckle when I saw that.